makefiles/murdock.inc.mk

```diff
@@ -0,0 +1,35 @@
+#
+# This file contains helper targets used by the CI.
+#
+
+# (HACK) get actual flash binary from FFLAGS.
+FLASHFILE:=$(filter $(HEXFILE) $(ELFFILE:.elf=.bin) $(ELFFILE),$(FFLAGS))
```

Comment:
It will not work for boards using `jlink.sh` as flasher as it does not get the filename from command line but from an environment variable… so there is nothing in FFLAGS.

Also having a `FLASHFILE` variable is something I am interested to fix in general.
Move the FLASHFILE outside of FFLAGS and be called as `$(FLASHER) $(FFLAGS) $(FLASHFILE)`.
We currently mix `FLASHFILE` and `HEXFILE/ELFFILE/BINFILE` names right now, and some HEXFILE are  `.bin`.

Also, for `nucleo` boards we generate only an `hex` file when a `.bin` file is needed to flash over the mounted volume. And we will need to be able to generate both outputs to handle different flashing.

Reply:
Yeah. I started fixing that for this PR, then decided that I'd prefer to support boards working with this FLASHFILE hack first and solve the problem afterwards. Ideally, once solved elsewhere, only the "FLASHFILE" variable has to be removed,and until then, we can run tests on all boards where the hack works.

Reply:
Yep, should be fixed in its proper PR.